src/libs/antares/study/include/antares/study/parts/hydro/container.h

```diff
@@ -167,10 +167,9 @@ class PartHydro
 // As this function can be called a lot of times, we pass working variables and returned variables
 // as arguments, so that we don't have to create them locally (as in a classical function) each
 // time.
-void getWaterValue(const double& level,
+double getWaterValue(const double& level,
```

Comment:
Took advantage of the hydro hot start removal to make some quick and small cleanings.